gprofiler/profilers/python_ebpf.py

```diff
@@ -0,0 +1,256 @@
+import glob
```

Comment:
Please add the project's license here :)

Reply:
Done